microshift_install_get_ready/images

```diff
@@ -0,0 +1 @@
+../images/
```

Comment:
These always confuse me, so please ignore if not applicable - just making sure that the relative links should only go up one - i.e., they should be `../images/` and not `../../images/`. (and attributes, snippets, etc)

That would be the case (I think) if the "get ready" folder is inside of the `microshift_install` folder, but I could be completely wrong here, since I'm not familiar with the structure, so please ignore if so.